lib/plugins/careportal.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+function init() {
+
+  var careportal = {
+    name: 'careportal'
+    , label: 'Care Portal'
+    , pluginType: 'drawer'
+  };
+
+  careportal.getEventTypes = function getEventTypes (sbx) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["sbx" is defined but never used](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2002418936/issues/source?bid=2452374&fileBranchId=3122262#l11)